src/NodeCollector/NodeAnalyzer/ArrayCallableMethodMatcher.php

```diff
@@ -162,6 +168,10 @@ private function resolveClassConstFetchType(ClassConstFetch $classConstFetch, Sc
             return new ObjectType($classConstantReference, null, $classReflection);
         }
 
+        if (is_string($classMethodName) && $classReflection->hasMethod($classMethodName)) {
```

Comment:
Could you use `hasNativeMethod()`? ensure to not rely on `@method` definition fallback when possible

Reply:
Thank you. Fixed.